### PR TITLE
RB-2.5 => RB-2.6 Merge fixes

### PR DESCRIPTION
**Pull request by rodlie**
Merge RB-2.5 into RB-2.6 with resolved conflicts. Please double check that everything is ok.

### CODE REVIEW

**Comment by acolwell** [comment]
LGTM

File: tools/MacPorts/graphics/openimageio/Portfile.rej [hunk 1]
```diff
@@ -0,0 +1,242 @@
+@@ -2,7 +2,6 @@
```
Comment (devernay) [inline tools/MacPorts/graphics/openimageio/Portfile.rej]: this file was committed by mistake, please remove

File: tools/MacPorts/graphics/openimageio/Portfile.rej.orig [hunk 2]
```diff
@@ -0,0 +1,234 @@
+@@ -17,7 +18,6 @@
```
Comment (devernay) [inline tools/MacPorts/graphics/openimageio/Portfile.rej.orig]: this file was committed by mistake, please remove

**Review by devernay — APPROVED** [review]
two extra files that could be removed, otherwise 👍

**Pull request merged**